contracts/ERC1155M.sol

```diff
@@ -368,7 +404,7 @@ contract ERC1155M is IERC1155M, ERC1155Supply, ERC2981, Ownable2Step, Reentrancy
         uint32 limit,
         bytes32[] calldata proof
     ) external payable onlyAuthorizedMinter {
-        _mintInternal(to, tokenId, qty, limit, proof);
+        _mintInternal(to, tokenId, qty, limit, proof, 0, bytes("0"));
```

Comment:
How will `assertValidCosign` pass for an `authorizedMinter` if the timestamp/signature is always empty? Would this mean the `authorizedMint` function will always fail if a `_cosigner` is set?

Reply:
`authorizedMint` will still work as Crossmint and Reservoir Relay allow us to pass arbitrary data, in this case, timestamp and signature.